content/en/docs/reference/glossary/resource-driver.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Resource Driver
+id: resource-driver
+date: 2023-10-16
+full_link: /docs/concepts/extend-kubernetes/compute-storage-net/resource-drivers/
+short_description: >
+  Software extensions to let Pods access devices that need vendor-specific initialization or setup
+aka:
+tags:
+- extension
+- fundamental
+---
+ Resource Driver is responsible for allocation of non-native resources requested by
```

Comment:
Why the capitalization? Should this be just "Resource driver"? Same below for "Resource Claim".

An article seems to be missing here. "A resource driver is ..."?

Reply:
https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects

Reply:
So "ResourceClaim" is okay, "Resource Claim" isn't.

Reply:
Fixed.